From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 5 Sep 2011 04:05:19 +0100
Subject: [PATCH 2/2] arm: ixp4xx, u300: Select ARCH_REQUIRE_GPIOLIB, not
 GENERIC_GPIO

The GPIO implementations for these two machines depend on gpiolib,
so they must not select GENERIC_GPIO directly.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 arch/arm/Kconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index f23712d..a1dbc4b 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -477,7 +477,7 @@ config ARCH_IXP4XX
 	depends on MMU
 	select CLKSRC_MMIO
 	select CPU_XSCALE
-	select GENERIC_GPIO
+	select ARCH_REQUIRE_GPIOLIB
 	select GENERIC_CLOCKEVENTS
 	select HAVE_SCHED_CLOCK
 	select MIGHT_HAVE_PCI
@@ -830,7 +830,7 @@ config ARCH_U300
 	select GENERIC_CLOCKEVENTS
 	select CLKDEV_LOOKUP
 	select HAVE_MACH_CLKDEV
-	select GENERIC_GPIO
+	select ARCH_REQUIRE_GPIOLIB
 	help
 	  Support for ST-Ericsson U300 series mobile platforms.
 
-- 
1.7.5.4

